docs/asciidoc/static/include/pluginbody.asciidoc

```diff
@@ -883,6 +941,17 @@ Then, you'll need to install your plugins dependencies with bundler:
 bundle install
 ----------------------------------
 
+[IMPORTANT]
+======
+
+If your plugin has an external file dependency described in `vendor.json`, you
+must download that dependency before testing:
```

Comment:
> must download that dependency before testing

"must download that dependency before testing or running. You can use `rake vendor` to do this.